cpp/oneapi/dal/algo/knn/common.hpp

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <bitset>
```

Comment:
Should it be included somewhere higher like detail common with creation of corresponding alias like it is done for shared ptr?